src/System.Private.CoreLib/src/System/Reflection/Associates.cs

```diff
@@ -131,11 +129,11 @@ internal static bool IncludeAccessor(MethodInfo associate, bool nonPublic)
                 Attributes.ComposedOfNoStaticMembers;
 
             while (RuntimeTypeHandle.IsGenericVariable(reflectedType))
-                reflectedType = (RuntimeType)reflectedType.BaseType;
+                reflectedType = (RuntimeType)reflectedType.BaseType!;
```

Comment:
if BaseType is nullable then this should be as well, right?

Reply:
`RuntimeTypeHandle.IsGenericVariable(reflectedType)` above doesn't accept nullalbe, so `reflectedType  `cannot be nullable